RecommendationEngineClient/RecommendationEngineClient/20 ClientOperations/Admin/AdminClientOperations.cs

```diff
@@ -0,0 +1,175 @@
+﻿using RecommendationEngineClient.Common.DTO;
+using RecommendationEngineClient.Common;
+using RecommendationEngineClient._10_Common;
+using RecommendationEngineClient._10_Common.DTO;
+
+namespace RecommendationEngineClient._20_ClientOperations.Admin
+{
+    public class AdminClientOperations : BaseClientOperations, IAdminClientOperations
+    {
+        public AdminClientOperations(RequestServices requestServices) : base(requestServices)
```

Comment:
do you need to explicitly define the empty constructor?

Reply:
Yes @chetanj257  In the client side, a class called RequestServices.cs is responsible for handling requests and responses. To streamline operations, a BaseClientOperations class has been created. This class contains a generic method that is inherited by all the services, including Chef, Admin, Employee, and others. To ensure efficiency and manage resources effectively, a single instance of RequestServices needs to be created when the Client is initiated. This instance should then be used throughout the entire lifespan of the client application.